docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 20th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using Fivetran and its [Stripe connector](https://fivetran.com/docs/applications/stripe). This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It also includes information regarding subscriptions and customers, as well as products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.
+
+Some of the customer information includes the customer's postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). However, as it intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it can be challenging to use to help on an individual customer level. Other uses for this data is for retrieving all-time counts (e.g., total number of active subscriptions).
+
+Fivetran preserves historical data within `subscription_history` by comparing the current state of the subscription with the most recent record in the history table and adding a new record if they vary.
+
+### Mobile app store providers
+Subscriptions Services data is also obtained from two mobile app store providers: Apple and Google. This includes date of billing and expired dates, subscription change event, etc. Apple provides limited data - it does not provide the country, price, or currency of subscriptions.
+
+Subscription Platform handles getting Apple subscriptions from the Apple API and stores the latest state of active subscriptions in Firestore. Prior to December 2022, VPN Apple subscriptions data was managed by VPN's Guardian service and stored in the Guardian database. As Firestore doesn't contain the full historical state of Apple subscriptions, Guardian data is still used for historical purposes.
+
+Similar to Apple, Subscription Platform also handles the current and historical state of Google subscriptions and it is synced to Firestore as well.
+
+The general use of this data is to understand what the subscription state was over time, as well as sequences of events.
+
+### FxA logs
+FxA logs include event data that can help provide insights regarding the customers' journey through the FxA sign-in/sign-up process. It can also help identify potential drivers (e.g., call-to-action, medium, paid search term, etc.) that directed customers to a page for acquisition and attribution purposes, if query parameters are set up properly (see below for more information).
```

Comment:
We have all kinds of user events here. Maybe also note account and subscription management events? E.g. we'll be implementing a subscription ended event as part of our current metrics epic: https://mozilla-hub.atlassian.net/browse/FXA-6955.